CONTRIBUTING.md

```diff
@@ -132,7 +132,8 @@ For model named `<main_model>` the following files may be created:
 * `templates/<main_model>.xml`
 * `views/<main_model>.xml`
 
-For `controller`, the only file should be named `main.py`.
+For `controller`, if there is only file should be named `main.py`. If there
```

Comment:
"if there is only file should be named `main.py`" should be "if there is only one file it should be named `main.py`"